### PR TITLE
Set up components

### PR DESCRIPTION
**Pull request by deyemiobaa**
set up components for the bokkstore app

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,12 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.1.1",
     "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-router-dom": "^6.3.0",
```
Comment (Cathella) [inline package.json:12]: ✅ `react-router-dom` is present in `package.json` 👍🏻.

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
-    </div>
-  );
+class AppBody extends React.PureComponent {
+  render() {
+    return (
+      <div className="container">
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/categories" element={<Categories />} />
+        </Routes>
```
Comment (Cathella) [inline src/App.js:14]: ✅ Has 2 `<Route>`s, one for the BookList/Books component and the other for the Categories component 👍🏻.

File: src/components/book/book-item.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ author, title }) => (
+  <li>
+    {author}
+    {' '}
+    {title}
+  </li>
```
Comment (Cathella) [inline src/components/book/book-item.js:9]: ✅ The Book component takes the book data as a prop and renders the title and the author 👍🏻

File: src/components/form/book-submit.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+              type="text"
+              className="form-control"
+              id="author"
+              placeholder="Author"
+            />
+          </div>
+          <button type="submit" className="btn btn-primary">
+            Add Book
+          </button>
+        </form>
```
Comment (Cathella) [inline src/components/form/book-submit.js:28]: ✅ Form component renders a form with inputs for the title and the author of the book 👍🏻 
✅ Has a button to submit 👍🏻

File: src/components/header/navigation.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+class NavBar extends React.PureComponent {
+  render() {
+    return (
+      <header className="navigation">
+        <h1>BookStore CMS</h1>
+        <nav className="nav-links">
+          <Link to="/" className="links">Books</Link>
+          <Link to="/categories" className="links">Categories</Link>
+        </nav>
```
Comment (Cathella) [inline src/components/header/navigation.js:13]: ✅ Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 👍🏻.

File: src/pages/books.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    <div className="book-list">
+      <ul>
+        <Book
+          author="J.K. Rowling"
+          title="Harry Potter and the Chamber of Secrets"
+        />
+      </ul>
+    </div>
+    <Form />
+  </div>
```
Comment (Cathella) [inline src/pages/books.js:18]: ✅ Renders a form by using the Form component 👍🏻

File: src/pages/categories.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import NavBar from '../components/header/navigation';
+
+const Categories = () => (
+  <div className="container">
+    <NavBar />
+  </div>
```
Comment (Cathella) [inline src/pages/categories.js]: ### 🛑 Required Changes
- [ ] Categories component is meant to display a button with text "Check status". Kindly fix this issue.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @deyemiobaa 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct GitHub Flow.
✅ Professional README.
✅ PR has a title and description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @deyemiobaa , 

Great work on making the changes requested by the previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**